.github/workflows/rust.yml

```diff
@@ -174,8 +174,8 @@ jobs:
   deploy-github-pages:
     needs: cargo-doc-artifact
     timeout-minutes: 5
-    # No point in deploying if we're not on `main`, as it will just fail.
-    if: github.ref == 'refs/heads/main'
+    # No point in deploying if we're not on `"stable"`, as it will just fail.
+    if: github.ref == 'refs/heads/"stable"'
```

Comment:
@neysofu is `stable` ok here?

Reply:
`stable` is fine, yes. We'll need @preston-evans98 to change the `gh-pages` deployment environment from `main` to `stable` as well. There's some docs here: https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#deployment-branches.

Reply:
I'll mark as resolved, I've created an issue to track this.

Reply:
I just noticed that `"stable"` is quoted here (both in the `github.ref` and the comment above). That will likely break the branch comparison.

Reply:
fixed